src/features/RandomMovie/RandomMovie.component.tsx

```diff
@@ -1,3 +1,59 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Container } from "react-bootstrap";
 
-export const RandomMovie = () => <div>RandomMovie</div>;
+import { Filters } from "features/Filters";
+import { SelectedFiltersType } from "features/Movies/types";
+import { EMPTY_STRING } from "constants/common";
+import { getDateString, getRandomMovieId } from "utils";
+import { useQuery } from "react-query";
+import { getMoviesByFilters } from "features/Movies/Movies.api";
+import { useAppSelector } from "hooks";
+import { MovieDetails } from "features/MovieDetails/MovieDetails.component";
+
+export const RandomMovie = () => {
+  const activeLanguage = useAppSelector((state) => state.language.activeLanguage);
+  const [startDate, setStartDate] = useState<any>(null);
+  const [endDate, setEndDate] = useState<any>(null);
+  const [genreId, setGenreId] = useState<string>(EMPTY_STRING);
+  const [selectedFilters, setSelectedFilters] = useState<SelectedFiltersType>(null);
```

Comment:
This states duplicate states from `src/features/Movies/Movies.component.tsx`, so may be we can move them into custom hook